### ISSUE TITLE
Sidekiq Web UI

### ISSUE THREAD

**Issue by brianpattison** [simplymadeapps/simple_scheduler#27]
We want to add a tab to the Sidekiq web UI to view a list of jobs scheduled using Simple Scheduler in a way that's easier to read than the existing scheduled jobs tab. There should also be a way to reset the scheduled jobs in the case of changes being made to the configuration file.

**Comment by brianpattison** [comment]
I started working on this again. Here's a little preview. Hopefully I can find some more time to knock out the rest of it soon.

![screen shot 2017-05-26 at 4 24 46 pm](https://cloud.githubusercontent.com/assets/124570/26513216/f54ec4ba-422f-11e7-97e4-23b50dcd0725.png)

**Comment by bmedenwald** [comment]
Neat!

-----------------------------
Brandon Medenwald
Co-Founder, Web Developer
Simply Made Apps

**Comment by brianpattison** [comment]
And here's a mockup of what I want to display after clicking on `Scheduled Jobs`:

![screen shot 2017-05-26 at 4 29 23 pm](https://cloud.githubusercontent.com/assets/124570/26513345/9888f4c0-4230-11e7-9275-bb7703f22225.png)

**Comment by bmedenwald** [comment]
What would "Add to queue" do?

**Comment by brianpattison** [comment]
@bmedenwald It'd be the same as the interface on the `Scheduled` tab, so these are the actual scheduled jobs, just filtered to the single Simple Scheduler "Task" as I call them, so it would run that scheduled job immediately.

**Comment by brianpattison** [comment]
@bmedenwald I could be convinced that it is not needed though.

**Comment by cuvant** [comment]
@brianpattison This is really nice! Do you plan a release in the nearby future?

**Comment by brianpattison** [comment]
@cuvant I honestly don't know if this is going to happen at all. Extending the Sidekiq web UI has been a bit of a nightmare, especially handling multiple versions on Sidekiq. If you have any experience extending Sidekiq's web UI, I sure could use some help! We don't have a real need for the web UI, so it hasn't been a priority.

Since we do have Rails as a gem dependency, I have thought about making Simple Scheduler a mountable Rails engine to add a web UI. This would be much easier than using Sidekiq's weird custom web app, but I would really like for it to integrate with Sidekiq's existing web UI.

**TL;DR:** Not in the near future. Maybe never. 🙁

**Comment by brianpattison** [comment]
I'm throwing in the towel on this for now. It really feels like I'm **hacking** on Sidekiq's web UI in my attempts to accomplish this, and I don't want to carry around my crappy code forever when it's not needed for this project to be useful.

If anyone in the community can point me to a good, clean example of integrating with Sidekiq's web UI that works with multiple versions on Sidekiq, I will revisit this project.
